api/tests/test_api_spec.py

```diff
@@ -0,0 +1,156 @@
+from conftest import AUTH_COOKIE, DEFAULT_USERNAME, JOBS_URI, auth, login, make_job, submit_batch
```

Comment:
importing `auth` from `conftest` threw me, think we should import it from `hyp3_api` where it's defined